src/api/handler/handler.js

```diff
@@ -1,17 +1,25 @@
-import {axiosNasaClient} from "../../clients/index.js";
+import {nasaAsteroidService} from "../../services/index.js";
 import {getPreviousPeriod, parseDate} from "./dateUtils.js";
 
+const parseQueryParams = (query) => {
+    return {
+        date: query.date,
+        countOnly: Boolean(query.count),
```

Comment:
Boolean will not work as you expect here, if query contains `count` with `'false'` value, `true` will be returned

Reply:
Yes, thank you for attentiveness. 
Fixed.